model/riscv_sys_control.sail

```diff
@@ -244,7 +244,20 @@ function getPendingSet(priv : Privilege) -> option((xlenbits, Privilege)) = {
     match processPending(mip, mie, mideleg.bits, mIE) {
       Ints_Empty()      => None(),
       Ints_Pending(p)   => Some((p, Machine)),
-      Ints_Delegated(d) => if sIE then Some((d, Supervisor)) else None(),
+      Ints_Delegated(d) => {
+        if not(extensionEnabled(Ext_S)) then {
+          // Can't delegate to user mode. This code is unreachable because
+          // `mideleg.bits` is 0 without supervisor mode.
```

Comment:
Well, the CSR doesn't exist so can't be written to by software and thus the Sail register remains its initial 0 value. The comment written here suggests that the CSR exists and is hardwired to 0 (which was the case in early specs).

Reply:
The Sail `register mideleg` always exists. That's what I was referring to. Not sure how to make that clearer...

Reply:
@jrtc27, @Timmmm : Can we mark this conversation as resolved?

Reply:
I'm going to merge this; happy to change the comment in future if desired.